src/index.js

```diff
@@ -9,16 +9,88 @@ import {
 import Customer from './Customer';
 import Hotel from './Hotel';
 
+
+//query selectors
+const totalSpent = document.querySelector('#totalSpent');
+const viewDescription = document.querySelector('#viewDescription');
+const mainSection = document.querySelector('#mainSection');
+const pastBookings = document.querySelector('#pastBookings');
+const currentBookings = document.querySelector('#currentBookings');
+
+//global variables
 let hotel, currentUser;
 
+
+//main startup function
 function startApplication() {
   Promise.all([getData('rooms'), getData('bookings'), getData('customers/1')]) //3rd request hardcoded for now
     .then(values => {
-      hotel = new Hotel(values[0], values[1])
+      hotel = new Hotel(values[0].rooms, values[1].bookings)
       currentUser = new Customer(values[2])
-      //dom display function kick-off here
+      populateLandingPage()
     })
 }
 
+function populateLandingPage() {
+  displayTotalCost()
+  displayCurrentBookings()
+}
+
+function displayTotalCost() {
+  const cost = hotel.calculateTotalCost(currentUser.id);
+  totalSpent.innerText = `Total spent on rooms: $${cost.toFixed(2)}`;
+}
+
+function displayCurrentBookings() {
+  const today = '2020/01/31'; //update this to be dynamic
```

Comment:
Date.now() gives the current date to make this dynamic if thats what you thinking but it does give a really crappy value that needs to be mutated. I have been told moment.js is super useful may be something to look into...